pkg/arvo/sys/vane/eyre.hoon

```diff
@@ -1725,6 +1726,9 @@
               duct-to-key.channel-state
             ?.  ?=(%| -.state.session)  duct-to-key.channel-state.state
             (~(del by duct-to-key.channel-state.state) p.state.session)
+          ::
+              connections
+            (~(del by connections.state) p.state.session)
```

Comment:
I suspect we should also produce a `[p.state.session %give %response %cancel ~]` move to make sure the request is known as "cancelled by arvo" in the runtime as well.

Reply:
Exciting! Perhaps we won't run out of file descriptors if we do this.

Reply:
vere no-ops on `%cancel`, and king-haskell doesn't even acknowledge the existence of the effect. We should have timeouts/process-limits such that the sockets and file descriptors are completely and correctly managed without Arvo's involvement. But implementing the effect could also spur such management.

/cc @pilfer-pandex

Reply:
In general, a runtime has to explicitly manage its resource utilization -- it can't depend on arvo to do that for it.